app/webpacker/lib/wca-data.js.erb

```diff
@@ -223,4 +223,5 @@ export const avatarImageTypes = <%= Rails.application.config.active_storage.web_
 
 // ----- TICKETS -----
 export const ticketTypes = <%= Ticket::TICKET_TYPES.to_json %>;
+export const ticketStatuses = <%= Ticket::TICKET_TYPES.transform_values { |value| value.constantize.statuses }.to_json %>;
```

Comment:
```suggestion
export const ticketStatuses = <%= Ticket::TICKET_TYPES.transform_values { |value| value.safe_constantize&.statuses }.to_json %>;
```

Reply:
Just in case one status isn't quite working as expected :wink:

Reply:
Thanks, done.